src/frontend/src/scheduler/task_context.rs

```diff
@@ -76,4 +77,8 @@ impl BatchTaskContext for FrontendBatchTaskContext {
     fn client_pool(&self) -> ComputeClientPoolRef {
         self.env.client_pool()
     }
+
+    fn get_config(&self) -> &BatchConfig {
+        todo!()
+    }
```

Comment:
These `todo!`s in `FrontendBatchTaskContext` looks so dangerous... @liurenjie1024